### PR TITLE
escp v2 tool

### PR DESCRIPTION
**Pull request by schmichael**
Out with the old and in with the new.

The old escp tool actually had some more features that we may want to re-implement, but the new tool doesn't OOM. The new code should also be mmmmmuch easier to work with than whatever I was doing before.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,303 +0,0 @@
...
-	}
-	fmt.Fprintf(os.Stderr, `Usage: 
-
-%s [options] \
-		http://localhost:9200/source_index \
-		http://locahost:9200/destination_index
-	  %s`, os.Args[0], "\n\n")
-	flag.PrintDefaults()
-	os.Exit(1)
-}
```
Comment (Ropes) [inline main.go]: The fact that this file goes byebye makes me happy!

Comment (schmichael) [inline main.go]: right? It was unsalvageable.

**Comment by Ropes** [comment]
Haven't totally read all the changes but it's way easier to follow thus far! :rocket:

**Comment by schmichael** [comment]
Merging because what really matters is the testing we're doing ourselves. :)

**Pull request merged**